tests/ValidatorTest.php

```diff
@@ -171,9 +169,7 @@ public function requiredDataProvider(): array
             'sort' => [
                 new InRange(
                     ['asc', 'desc'],
-                    skipOnEmpty: static function (mixed $value, bool $isAttributeMissing): bool {
-                        return $isAttributeMissing || $value === '';
-                    }
+                    skipOnEmpty: static fn (mixed $value, bool $isAttributeMissing): bool => $isAttributeMissing || $value === ''
```

Comment:
This one is too long now. We actually decided to disable this rule completely because it also transforms multiple line static functions like that. Ideally need to configure it to transform only when the result fits max line length.

Can be disabled like that - https://github.com/yiisoft/yii-runner-console/pull/22/commits/ff8151d696cad3464974fa6da99e52fb78acfc11.

Reply:
Fix line lenght in this case.